src/components/Filter.tsx

```diff
@@ -26,38 +27,55 @@ export default class Filter extends React.Component<FilterProps, FilterState> {
   constructor(props: FilterProps) {
     super(props);
     this.flip = this.flip.bind(this);
-    this.state = { flip: this.props.initialFlip };
+    this.state = { flip: this.props.initialFlip, panelOpen: false };
```

Comment:
This should follow a similar pattern to the `flip` key and the `initialFlip` prop. So instead of `false`, it should be `this.props.panelOpen` and then set it to `false` by default.

Reply:
Actually, I take this back. This is the _state_ and not a _prop_. So this is fine to leave as is.